game.py

```diff
@@ -12,7 +12,17 @@ class Random(Bot):
     """
     Cory's bot :)
     """
-    pass
+
+    def move(self, state):
+        valid_pile_indexes = []
+        for i, pile in enumerate(state):
+            if pile >=1:
+                valid_pile_indexes.append(i)
+        if not valid_pile_indexes:
+            raise NimException(f'No valid moves on state: {state}')
+        pile = choice(valid_pile_indexes)
```

Comment:
Here `pile` represent the index of a pile while above it represented the number of stones in a pile. You should be consistent to avoid confusion.